apis/config/v1beta1/configuration_types.go

```diff
@@ -387,6 +387,25 @@ type Integrations struct {
 	LabelKeysToCopy []string `json:"labelKeysToCopy,omitempty"`
 }
 
+type KueueIntegrations string
```

Comment:
maybe IntegrationName. I think putting  Kueue into the enum name is not necessary as it is already in the package naming. It shohld also be singular

Reply:
Alternatively FrameworkName as this is how we name the vars currently

Reply:
I'll wait for agreement on @tenzen-y as I will have to change that name everywhere.

I am fine with FrameworkName.

Reply:
I prefer `FrameworkName`

Reply:
Maybe actually IntegrationReference, as this is under "Integrations" key, and we also add "Reference" for other such string aliases, like TopologyReference, etc.

Reply:
Let's describe what are these,

> Kueue has "JobFramework", and each "Integration"s (Job, JobSet...) can implement the JobFramework and support Kueue enhancement.

> Kueue has "JobFramework", and each "Framework"s (Job, JobSet...) can implement the JobFramework and support Kueue enhancement.


It seems that "Integration" is more appropriate.

Reply:
> So maybe "IntegrationReference", for consitency?

SGTM

Reply:
I'm chasing one unit test failure but once that is debugged, I'll pusha  change to rename to IntegrationReference.

Reply:
Renamed to IntegrationReference and I fixed the unit test failure!